packages/swapper/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# [@shapeshiftoss/swapper-v16.0.0](https://github.com/shapeshift/lib/compare/@shapeshiftoss/swapper-v15.8.0...@shapeshiftoss/swapper-v16.0.0) (2023-01-30)
```

Comment:
no need to do this manually - semantic release generates this automatically from the commit messages into `main`

Reply:
ah this is just a rebase issue nvm